source/docs/articles/sites/code/redirect-incoming-requests.md

```diff
@@ -30,9 +30,9 @@ Automatic resolution of domains is not supported. For each domain that you want
 
 ## Redirect to a Common Domain
 
-While it’s good for visitors and DNS to resolve both www and the domain itself, it's best practice to choose one or the other and redirect from www to non-www (or vice versa, your call). This optimizes SEO by avoiding duplicate content and prevents session strangeness, where a user can be signed on one domain but logged out of other domains at the same time.  Choose one of the blocks of code below:
+While it’s good for visitors and DNS to resolve both www and the domain itself, it's best practice to choose one or the other and redirect from www to non-www (or vice versa, your call). This optimizes SEO by avoiding duplicate content and prevents session strangeness, where a user can be signed on one domain but logged out of other domains at the same time.  Choose the following block to standardize on the www domain:
 ```
-    // Require WWW.
+    // Redirect all traffic to WWW. For example: www.yoursite.com
```

Comment:
Isn't this only necessary for `https` traffic? the dub-dub-dubber does this already.